lib/performance_data/statistics.rb

```diff
@@ -0,0 +1,216 @@
+require 'json'
+require 'gds_api/performance_platform/data_out'
+
+module PerformanceData
+  class Statistics
+    attr_accessor :data_out, :content, :slug, :part_urls, :document_type, :is_multipart, :performance_data
+
+    def initialize(content, slug)
+      @data_out = GdsApi::PerformancePlatform::DataOut.new("http://www.performance.service.gov.uk")
+      @content = content
+      @slug = slug
+      @part_urls = get_part_urls(content, slug)
+      @document_type = @content["document_type"] || @content["format"]
+      @is_multipart = @part_urls.any? || (document_type == 'smart-answer')
```

Comment:
Yeah, I think that's why we have to special case the "is it multipart" code to allow for them as the smart answers schemas (for whatever reason) don't have the same part structure as the other multipart schemas.  Thanks for looking into it though; sounds like there's nothing extra for us to do here though so that's good.